src/App.js

```diff
@@ -33,12 +46,14 @@ function App() {
           <Route path ="/" element={ 
             <Fishes fishes={fishes}/>
           } />
```

Comment:
error and setError props are passed down to the <Focus> component correctly, which can use them for error handling when fetching data from the API or dealing with other errors. The Routes seem to implemented correctly

If im correct it seems the the last <Route path="*"> handles unknown routes and redirects them to the 404 page using the <Navigate> component.?

Reply:
Yea! The last routes redirects the users to a 404 error page whenever the user somehow reaches an unknown route.